packages/frontend/src/Build.spec.ts

```diff
@@ -548,7 +545,7 @@ test('Show the image if isManifest: true and Labels is empty', async () => {
   render(Build, { imageName: 'testmanifest1', imageTag: 'latest' });
 
   await vi.waitFor(() => {
-    expect(spyOnInspectManifest).toHaveBeenCalled();
+    expect(vi.mocked(bootcClient.inspectManifest)).toHaveBeenCalled();
```

Comment:
I think you can simplify with:
```
expect(bootcClient.inspectManifest).toHaveBeenCalled();
```

Reply:
I was trying to clean up one bad pattern and copied another one 😬. Fixed these in new commit, including all the other instances in this repo.